tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -2533,9 +2753,9 @@ lldbg_fwd_local_variable(LL_DebugInfo *db, int sptr, int findex,
    \param findex  Pass through argument for creating forward reference
    \param db      The debug info
  */
-INLINE static void
-init_subrange_bound(LL_DebugInfo *db, ISZ_T *cb, LL_MDRef *bound_sptr,
-                    SPTR sptr, ISZ_T defVal, int findex)
+INLINE static void init_subrange_bound_pre11(LL_DebugInfo *db, ISZ_T *cb,
```

Comment:
FWIW, the PGI coding standard was to keep the function names starting in column 0.

Reply:
Thanks for this. I have corrected this.
Now I realize it is due to tool "git-clang-format" which I use, is there any equivalent tool for PGI coding style ?